docs/source/tutorials/odsc_tutorial/remote_local.rst

```diff
@@ -0,0 +1,74 @@
+Launching Experiments Remotely
+==============================
+
+As a machine learning practitioner, you operate in a highly competitive
+landscape. Your success depends to a large extent on whether you can *decrease
+the time to the next decision*. In this section, we discuss one important
+approach, namely how to increase the *number of experiments run in parallel*.
+
+Launching our Study
+-------------------
+
+Here is how we specified and ran experiments of the
+`study above <comparison.html#a-comparative-study>`__. First, we specify a
```

Comment:
Nit: this is on a new page, so for a viewer looking at the rendered docs, they won't see it above, they will see it on the previous page